tox.ini

```diff
@@ -1,20 +1,20 @@
 [tox]
 envlist =
-    py39-django{22,30,31,32,40},
-    py{37,38,310}-django{32},
+    py{37,38,39}-django{22,30,31}
+    py{37,38,39,310}-django32,
+    py{38,39,310}-django40,
     flake8,doclint
 
 [testenv]
 usedevelop = True
 pip_pre = False
 extras = dev
 basepython =
-    py36: python3.6
     py37: python3.7
     py38: python3.8
     py39: python3.9
     py310: python3.10
-    serve: python3.6
+    serve: python3.7
     docs: python3.10
     doclint: python3.10
```

Comment:
I'd tend to say "python3" for those and rely on the default version of the machine using it, so we don't have to update it.